### PR TITLE
Allow 2018 boilerplate

### PR DESCRIPTION
**Pull request by fejta**
/assign @ixdy @cjwagner

### CODE REVIEW

File: hack/verify_boilerplate.py [hunk 1]
```diff
@@ -154,7 +154,7 @@ def get_regexs():
     # Search for "YEAR" which exists in the boilerplate, but shouldn't in the real thing
     regexs["year"] = re.compile('YEAR')
     # dates can be 2014, 2015, 2016 or 2017, company holder names can be anything
```
Comment (ixdy) [inline hack/verify_boilerplate.py]: comment needs updating ;)

File: hack/verify_boilerplate.py [hunk 2]
```diff
@@ -154,7 +154,7 @@ def get_regexs():
...
-    regexs["date"] = re.compile('(2014|2015|2016|2017)')
```
Comment (ixdy) [inline hack/verify_boilerplate.py]: I wonder if this gives any more value over something more flexible like `\d{4}`...

File: hack/verify_boilerplate.py (hunk 1 above)
Comment (fejta) [reply hack/verify_boilerplate.py]: You mean deleting?

**Pull request merged**